cfme/ansible/credentials.py

```diff
@@ -393,7 +393,7 @@ class Credential(BaseEntity, Taggable):
     # is also problematic for attrs
 
     def __init__(self, collection, name, credential_type, **credentials):
-        super(Credential, self).__init__(collection)
+        super().__init__(collection)
```

Comment:
:+1:

Reply:
Yeah this is part of the pyupgrade conversion, not something I changed directly.